spec/models/comment_spec.rb

```diff
@@ -1,5 +1,31 @@
-require 'rails_helper'
+require "rails_helper"
 
 RSpec.describe Comment, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  let(:user) { build(:user) }
+  subject { build(:comment, user: user) }
+
+  context "when the factory is build" do
+    it { is_expected.to be_valid }
+    it { is_expected.to be_a Comment }
+  end
+
+  context "when calling the instance methods" do
+    it do
+      is_expected.to respond_to(
+        :user_id, :content, :comment_on_id, :comment_on_type, :votes_count
+      )
```

Comment:
You could extend this to include other methods created by the various relationships such as:
`user`, `answer`, `question`, `votes` etc